src/frontend/apps/impress/src/cunningham/cunningham-style.css

```
@@ -1,4 +1,4 @@
-@import url('@gouvfr-lasuite/ui-kit/style');
+@import url('@gouvfr-lasuite/ui-components/style');
```

Stylelint (17.14.0)

[error] Expected "url('`@gouvfr-lasuite/ui-components/style`')" to be "'`@gouvfr-lasuite/ui-components/style`'" (import-notation)

(import-notation)